workflow/applications/applications.py

```diff
@@ -32,57 +31,50 @@ def __init__(self, conf: Configuration) -> None:
 
         self.scheduler = Host().scheduler
 
-        # Save the configuration so we can source the config files when
-        # determining task resources
-        self.conf = conf
+        self._base = conf.parse_config('config.base')
```

Comment:
Don't like making this an object field. This version should not be needed beyond initialization. All later uses should be using `self.configs[run]['base']`.
```suggestion
        base = conf.parse_config('config.base')
```

Reply:
Alright, I removed the `_base` attribute from the `AppConfig` class and subclasses.  `_no_run` is no longer referenced anywhere outside of `_init_finalize`.  This is still required to determine which `run`s are valid for the experiment.